### ISSUE TITLE
Please include support for mysql or postgres Data format

### ISSUE THREAD

**Issue by s1-ranjan** [nicktylah/gnucash-mobile#23]
It would be nice if the mobile version had support for MySQL & Postgres data format, then it can be very helpful for data sync for numerous devices and/or on the go.

**Comment by aekramer** [comment]
Agreed, I use  gnucash currently on a server with mysql access (from pc). It would be great to get access to the same data from anywhere.
